docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description:** The minimum time interval that a Tablet Compaction can be scheduled since the last compaction failure.
+
+#### max_compaction_concurrency
+
+- **Default:** -1
+- **Description:** The maximum concurrency of compactions (both Base Compaction and Cumulative Compaction). The value -1 indicates that no limit is imposed on the concurrency.
+
+#### periodic_counter_update_period_ms
+
+- **Default:** 500 ms
+- **Description:** The time interval at which to collect the Counter statistics.
+
+#### load_error_log_reserve_hours
+
+- **Default:** 48 hours
+- **Description:** The time for which data loading logs are reserved.
+
+#### streaming_load_max_mb
+
+- **Default:** 10,240 MB
+- **Description:** The maximum size of a file that can be streamed into StarRocks.
+
+#### streaming_load_max_batch_size_mb
+
+- **Default:** 100 MB
+- **Description:** The maximum size of a JSON file that can be streamed into StarRocks.
+
+#### memory_maintenance_sleep_time_s
+
+- **Default:** 10 seconds
+- **Description:** The time interval at which ColumnPool GC is triggered. StarRocks executes GC periodically and returns the released memory to the operating system.
+
+#### write_buffer_size
+
+- **Default:** 104,857,600 Bytes
+- **Description:** The buffer size of MemTable in the memory. This configuration item is the threshold to trigger a flush.
+
+#### tablet_stat_cache_update_interval_second
+
+- **Default:** 300 seconds
+- **Description:** The time interval at which to update Tablet Stat Cache.
+
+#### result_buffer_cancelled_interval_time
+
+- **Default:** 300 seconds
+- **Description:** The wait time before BufferControlBlock releases data.
+
+#### thrift_rpc_timeout_ms
+
+- **Default:** 5,000 ms
+- **Description:** The timeout for a thrift RPC.
+
+#### txn_commit_rpc_timeout_ms
+
+- **Default:** 60,000 ms
+- **Description:** The timeout for a transaction commit RPC.
+
+#### max_consumer_num_per_group
+
+- **Default:** 3 (Maximum Number of Consumers in a Consumer Group of Routine Load)
+- **Description:** The maximum number of consumers in a consumer group of Routine Load.
+
+#### max_memory_sink_batch_count
+
+- **Default:** 20 (Maximum Number of Scan Cache Batches)
+- **Description:** The maximum number of Scan Cache batches.
+
+#### scan_context_gc_interval_min
+
+- **Default:** 5 minutes
+- **Description:** The time interval at which to clean the Scan Context.
+
+#### path_gc_check_step
+
+- **Default:** 1,000 (Maximum Number of Files Scanned Continuously Each Time)
+- **Description:** The maximum number of files that can be scanned continuously each time.
+
+#### path_gc_check_step_interval_ms
+
+- **Default:** 10 ms
+- **Description:** The time interval between file scans.
+
+#### path_scan_interval_second
+
+- **Default:** 86,400 seconds
+- **Description:** The time interval at which GC cleans expired data.
+
+#### storage_flood_stage_usage_percent
+
+- **Default:** 95% (Storage Usage Threshold for Load and Restore Job Rejection)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8d653a15-805b-46db-86fe-ddf882885606)
All these should be deleted.